scripts/files/notarize_config.json.tmpl

```diff
@@ -0,0 +1,10 @@
+{
+  "apple_id": {
+    "username" : "{{APPLE_ID_USERNAME}}",
+    "password":  "{{APPLE_ID_KEY}}"
+  },
+  "notarize" : [{
+    "path": "percy-macos.zip",
+    "bundle_id": "percy.io.cli"
```

Comment:
```suggestion
    "bundle_id": "io.percy.cli"
```

not sure if this java style package name?, coz - `cli.percy.io` entails to `io.percy.cli` package.